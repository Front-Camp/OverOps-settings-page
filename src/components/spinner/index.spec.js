import React from 'react';
import Spinner from './index';
import {shallow} from 'enzyme/build';

describe('Spinner', () => {
  it('Component should be rendered correctly', () => {
    const component = shallow(<Spinner />);

    // Snapshot demo
    expect(component).toMatchSnapshot();
  });
});
