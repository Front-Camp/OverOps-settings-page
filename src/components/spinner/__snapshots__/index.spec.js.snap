// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component should be rendered correctly 1`] = `
<div
  className="spinner"
/>
`;

exports[`Spinner should rendered correctly 1`] = `
<div
  className="spinner"
/>
`;
